services/motion/builtin/builtin_test.go

```diff
@@ -380,7 +380,7 @@ func TestObstacleReplanningGlobe(t *testing.T) {
 				caseName := "test-case-3"
 				// This base will always see an obstacle 800mm in front of it, triggering several replans.
 				// However, enough replans should eventually get it to its goal.
-				obstaclePosition := spatialmath.NewPoseFromPoint(r3.Vector{X: 800, Y: 0, Z: 0})
+				obstaclePosition := spatialmath.NewPoseFromPoint(r3.Vector{X: 500, Y: 0, Z: 0})
```

Comment:
this is now a flaky test. TODO:  := fix
passes on local machine with local architecture.